x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -132,13 +290,13 @@ export const registerCollector: RegisterCollector = ({
         },
       },
     },
-    isReady: () => kibanaIndex.length > 0,
+    isReady: () => kibanaIndex.length > 0 && signalsIndex.length > 0,
```

Comment:
This feels odd to me. Won't the kibanaIndex length and signalsIndex always be greater than zero since they're just configurations of strings that always have defaults? I wonder if this old left over logic that is being added to that doesn't really add anything as both of these are alway true.

Reply:
I think that's an accurate assessment @FrankHassanabad; I believe this function is (was?) used by other collectors that do pre-processing of their data to determine whether the next batch of processing has been completed. We shouldn't need more than `() => true` here.

Reply:
Thanks! Updated in 0c91a78